src/main/java/com/getindata/connectors/http/HttpSinkBuilder.java

```diff
@@ -70,6 +70,8 @@
     // If not defined, should be set to DEFAULT_CLIENT_BUILDER
     private SinkHttpClientBuilder sinkHttpClientBuilder;
 
+    private HttpSinkPostRequestCallback httpSinkPostRequestCallback;
```

Comment:
what if this be null?
I mean what if user will not set the callback via builder in streaming API?

Maybe we should have default value assigned like we have for http client builder ?